### DISCUSSION TITLE
upgrade from rook v1.8.1 to v1.11.1

### DISCUSSION THREAD

**Discussion by vadlakiran** [rook/rook#12035] [General]
**Is this a bug report or feature request?**
* Bug Report
i would like to understand the insted of upgrading increament level like v1.8.1 to 1.9.13 , want to upgrade directly from v1.8.1 to v1.11.1
**Deviation from expected behavior:**

**Expected behavior:**

**How to reproduce it (minimal and precise):**

**File(s) to submit**:

* Cluster CR (custom resource), typically called `cluster.yaml`, if necessary

**Logs to submit**:

* Operator's logs, if necessary
* Crashing pod(s) logs, if necessary

  To get logs, use `kubectl -n <namespace> logs <pod name>`
  When pasting logs, always surround them with backticks or use the `insert code` button from the Github UI.
  Read [GitHub documentation if you need help](https://help.github.com/en/articles/creating-and-highlighting-code-blocks).

**Cluster Status to submit**:

* Output of krew commands, if necessary

  To get the health of the cluster, use `kubectl rook-ceph health`
  To get the status of the cluster, use `kubectl rook-ceph ceph status`
  For more details, see the [Rook Krew Plugin](https://rook.io/docs/rook/latest-release/Troubleshooting/krew-plugin)

**Environment**:
* OS (e.g. from /etc/os-release): ubuntu
* Kernel (e.g. `uname -a`):
* Cloud provider or hardware configuration:
* Rook version (use `rook version` inside of a Rook Pod): v1.8.1
* Storage backend version (e.g. for ceph do `ceph -v`): 16.2.7
* Kubernetes version (use `kubectl version`): v1.21.5
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift):
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/latest-release/Troubleshooting/ceph-toolbox/#interactive-toolbox)):

**Comment by parth-gr** [comment]
I don't think so you can jump version ,
refer https://rook.io/docs/rook/v1.11/Upgrade/rook-upgrade/
cc @travisn

**Reply by galexrt** [reply]
Yes, Rook minor versions shouldn't be jumped. So you must go from 1.8.1 to the latest 1.8.x patch release, from 1.8.x to 1.9.x, 1.9.x to 1.10.x and so on.

Why? Because Rook only tests upgrading from one minor version to the next. So it could work but the project recommendation is to go through all minor updates one by one as shown in the upgrade guide.

Note: I converted the issue to a discussion.

**Reply by travisn** [reply]
I'll just add that while it may be possible to jump versions since we haven't had any bigger breaking changes for a few releases, I want to emphasize that you will need to take the burden of testing this. Rook maintainers only support the upgrade between each version due to the complexity of the test matrix and dealing with upgrade issues when skipping versions.

**Reply by vadlakiran** [reply]
ok, Thank you
expecting that we can upgrade the v1.8.1 to any version. so its save the time .
